src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/integration/LicenceIntegrationTest.kt

```diff
@@ -680,7 +680,44 @@ class LicenceIntegrationTest : IntegrationTestBase() {
     assertThat(result?.topupSupervisionStartDate).isEqualTo(LocalDate.parse("2024-09-11"))
     assertThat(result?.topupSupervisionExpiryDate).isEqualTo(LocalDate.parse("2025-09-11"))
   }
+  @Test
+  @Sql(
+    "classpath:test_data/seed-licence-id-3.sql"
+  )
+  fun `Update sentence dates should set license status to inactive when the offender has a new future release date`() {
+    prisonApiMockServer.stubGetHdcLatest()
+
+    webTestClient.put()
+      .uri("/licence/id/3/sentence-dates")
+      .bodyValue(
+        UpdateSentenceDatesRequest(
+          conditionalReleaseDate = LocalDate.now().plusDays(5),
+          actualReleaseDate = LocalDate.now().plusDays(2),
+          sentenceStartDate = LocalDate.parse("2021-09-11"),
```

Comment:
I'd be inclined to have any future dates as relative to LocalDate.now() like you've done on L695, just to prevent tests filing unexpectedly when the future dates have been reached.